nifi-nar-bundles/nifi-site-to-site-reporting-bundle/nifi-site-to-site-reporting-task/src/main/resources/docs/org.apache.nifi.reporting.SiteToSiteStatusReportingTask/additionalDetails.html

```diff
@@ -50,7 +50,7 @@ <h2>Record writer</h2>
 	{ "name" : "actorHostname", "type" : "string"},
 	{ "name" : "componentType", "type" : "string"},
 	{ "name" : "componentName", "type" : "string"},
-	{ "name" : "parentId", "type" : "string"},
+	{ "name" : "parentId", "type" : ["string", "null"]},
```

Comment:
Isn't there some Avro bug where the null has to come first? If that's been fixed, I'm good with this, otherwise we might have to switch it around.

Reply:
If you mean [this bug](https://issues.apache.org/jira/browse/AVRO-1118) looks like it was fixed a long time ago.

Reply:
That looks VERY familiar but I think even the first time we dealt with Avro it might have been 1.7.x after it had been fixed (plus that had to do with default values being supported with unions). Maybe I'm thinking of a NiFi Record API bug with unions and defaults. But either way, if this LGTY it LGTM :)

Reply:
I think I remember vaguely something like that but I think it was an issue on our side with the Record API (can't find a JIRA though). I looked at other projects I've been working on and I've schemas with "null" being first or second and I had no issue.